deepinv/datasets/__init__.py

```diff
@@ -4,11 +4,12 @@
 from .urban100 import Urban100HR
 from .set14 import Set14HR
 from .cbsd68 import CBSD68
-from .fastmri import (
+from .mri.fastmri import (
     FastMRISliceDataset,
     SimpleFastMRISliceDataset,
     MRISliceTransform,
 )
+from .mri.calgary import Calgary3DBrainMRIDataset, CalgaryDataTransformer
```

Comment:
I think `CalgaryDataTransform` would be better to be similar to fastMRI `MRISliceTransform`